apps/web/modules/api/management/auth.ts

```
@@ -0,0 +1,147 @@
+  authentication: TAuthenticationApiKey;
+  environmentId: string;
+}): Result<void, ApiErrorResponse> => {
+  if (authentication.type === "apiKey" && authentication.environmentId !== environmentId) {
+    return err({
+      type: "unauthorized",
+    });
+  }
+  return okVoid();
+};
+
+export type HandlerFn<TInput = {}> = ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)